### PR TITLE
Updating devops.md

### PR DESCRIPTION
**Pull request by pallayr**
Adding more sales pitch and verbage

- [x] Ready for review
- [x] Ready for merge

### CODE REVIEW

File: devops.MD [hunk 1]
```diff
@@ -1,32 +1,32 @@
 # Introduction
-At The Startup Factory we are primarily interested in small-scale developments that are meant to test business hypotheses.
+At The Startup Factory, we thrive to help small-scale developments which can be engaged to valid business hypotheses and used to make effective decisions.
```
Comment (pallayr) [inline devops.MD]: Probably need breaking up and reworded

Comment (archena) [reply devops.MD]: 'thrive on helping'?

Comment (archena) [reply devops.MD]: validate

File: devops.MD [hunk 2]
```diff
@@ -1,32 +1,32 @@
...
+Our processes are designed to be simple, repeatable and based upon open tools we have practical experience with and know well. We have the appetite and the motivation to try new things, being at the cutting edge so that your ideas don’t lose impact with the best technology. These guidelines are a starting point serving to fulfil the former need, but still allowing us to pursue the latter - technology choices are not as fixed as the core approaches are.
 
 # We value deployments that are
 * Automated
-  * By codifying processes as executable code stored in version control
+  * _Codifying_ processes as code that can be run and stored in version control
 
 * Parameterised
-  * By exposing configuration as environment variables
+  * Exposing configuration as variables which can be alter per environment
```
Comment (archena) [inline devops.MD]: altered

File: devops.MD [hunk 3]
```diff
@@ -1,32 +1,32 @@
...
-  * By avoiding side effects, non-codified processes, and unmanaged state
+  * Avoiding side effects, adhoc processes, or unmanaged state
 
 * Immutable
-  * By containerisation
+  * Containerising and keeping the images in a well-known state
 
 # Cloud providers
-We prefer AWS but might use Heroku for quick proofs-of-concept.
+Our solutions can be cloud-agnostic but we prefer AWS and have been using it before it became fashionable. We also use other platforms such as Heroku for quick proofs-of-concept tii.
```
Comment (archena) [inline devops.MD]: using it *since* before

Comment (archena) [reply devops.MD]: tii?

File: devops.MD [hunk 4]
```diff
@@ -1,32 +1,32 @@
...
-  * By avoiding side effects, non-codified processes, and unmanaged state
+  * Avoiding side effects, adhoc processes, or unmanaged state
 
 * Immutable
-  * By containerisation
+  * Containerising and keeping the images in a well-known state
 
 # Cloud providers
-We prefer AWS but might use Heroku for quick proofs-of-concept.
+Our solutions can be cloud-agnostic but we prefer AWS and have been using it before it became fashionable. We also use other platforms such as Heroku for quick proofs-of-concept too.
```
Comment (pallayr) [inline devops.MD:20]: Using `before` is intentionally: Cake's been using AWS before it became in vogue and everyone else started to use it.

**Pull request merged**